aw/ViewControllers/Reaches/RunListTableViewController.swift

```diff
@@ -30,6 +30,18 @@ class RunListTableViewController: UIViewController, MOCViewControllerType {
         updateTime()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        guard let lastUpdated = DefaultsManager.lastUpdated else {
+            refreshData()
+            return
+        }
+        if lastUpdated < Date().addingTimeInterval(-3600) {
```

Comment:
Let's pull the update interval into a constant with the units in hungarian, like UPDATE_INTERVAL_s.